packages/dds/tree/docs/user-facing/schema-evolution.md

```diff
@@ -0,0 +1,304 @@
+# Schema Evolution
+
+This document outlines the general constraints around maintaining compatibility in an application document ecosystem in the face of schema changes.
+
+## Context
+
+When an application creates and uses a schema to open a document using `SchemaFactory` and `TreeView.initialize`,
+information about that schema is persisted as part of the document.
+The schema that the application works with in code is hereforth referred to as "view schema" and the schema persisted with the document "stored schema."
+
+Compatibility scenarios can be described in terms of discrepancies between the view schema and the stored schema.
```

Comment:
Are we still planning to support something like differences to track the actual mismatches between schemas, similar to the contents within `treeDiscrepancies.ts` in [v2](https://github.com/microsoft/FluidFramework/blob/b2f9c65b29d555c0b299356ba004d35fa220cb14/packages/dds/tree/src/simple-tree/treeDiscrepancies.ts)? It seems that the definition of the discrepancy has changed.